src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.js

```diff
@@ -158,8 +158,11 @@ module.directive('vislibValueAxes', function () {
             label = matchingSeries[0].makeLabel();
           }
           if (lastAxisTitles[axis.id] !== label && label !== '') {
+            // Only overwrite the custom title with the value axis label if it hasn't been changed
+            if (lastAxisTitles[axis.id] === axis.title.text) {
```

Comment:
wouldn't we want to change the title automatically if the actual aggregation was changed, or a field ?
- select sum over bytes field
- go to axes, change title from sum of bytes to `sum of bytes downloaded`
- go back to data tab, and change field to something else ... what would you expect in this case ? 
- what about if you completely changed aggregation ?

Reply:
@ppisljar Good catch -- it feels to me the expected behavior in the cases you've described would be that the custom title gets cleared when the agg type or field changes. I will give it some thought and follow up in a bit.